Mathlib/Analysis/Normed/Ring/Seminorm.lean

```diff
@@ -149,6 +150,19 @@ theorem seminorm_one_eq_one_iff_ne_zero (hp : p 1 ≤ 1) : p 1 = 1 ↔ p ≠ 0 :
 
 end Ring
 
+section CommRing
+
+variable [CommRing R] (p : RingSeminorm R)
+
+theorem exists_index_pow_le (hna : IsNonarchimedean p) (x y : R) (n : ℕ) :
+    ∃ (m : ℕ) (_ : m ∈ Finset.range (n + 1)), p ((x + y) ^ (n : ℕ)) ^ (1 / (n : ℝ)) ≤
```

Comment:
```suggestion
    ∃ m : ℕ, m ∈ Finset.range (n + 1) \and p ((x + y) ^ (n : ℕ)) ^ (1 / (n : ℝ)) ≤
```
though is there a reason to use `range` here rather than just `<`?